CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 ## 0.2.0
 
-* Added `Control.Applicative.Combinators.NonEmpty` module that exports
```

Comment:
Well that was a complete mess. Sorry the history is _way more_ convoluted than it needs to be.

If you know of some way to rewrite the history to make things clearer, please feel free.